packages/dappmanager/src/daemons/telegram/telegramCommands.ts

```diff
@@ -0,0 +1,127 @@
+          const channelExists = checkIfChannelIdExists(chatId);
+          if (channelExists === false) {
+            const message = buildTelegramMessage({
+              telegramMessage: "Succesfully saved channel ID",
+              telegramMessageType: "Success"
+            });
+            subscribeChannelId(chatId);
+            await sendTelegramMessage({ bot, chatId, message });
+          }
+        } catch (e) {
+          logs.error("Error on telegram daemon. message polling", e);
+        }
+    });
+
+    // Remove channel ID
+    bot.onText(/\/unsubscribe/, async msg => {
+      try {
+        const chatId = msg.chat.id.toString();
+        let message = "";
+        const channelExists = checkIfChannelIdExists(chatId);
+        if (channelExists === true) {
+          unsubscribeChannelId(chatId);
+          message = buildTelegramMessage({
+            telegramMessage: "Succesfully removed channel ID",
+            telegramMessageType: "Success"
+          });
+        } else {
+          message = buildTelegramMessage({
+            telegramMessage: "Channel Id not found",
+            telegramMessageType: "Danger"
+          });
+        }
+
+        await sendTelegramMessage({ bot, chatId, message });
+      } catch (e) {
+        logs.error("Error on telegram daemon. /unsubscribe command", e);
+      }
+    });
+
+    // Returns help content
+    bot.onText(/\/help/, async msg => {
+      try {
+        const chatId = msg.chat.id.toString();
+        const message = buildTelegramMessage({
+          telegramMessage: `${bold("Commands")}:\n
+           ${bold("/unsubscribe")} unsubscribes the channel\n
+           ${bold("/help")} returns help content\n
+            More information ${url(
+              "here",
+              "https://hackmd.io/iJngUGVkRMqxOEqFEjT0XA"
+            )}`,
+          telegramMessageType: "Help"
+        });
+
+        await sendTelegramMessage({ bot, chatId, message });
+      } catch (e) {
+        logs.error("Error on telegram daemon. /help command", e);
+      }
+    });
+  } catch (e) {
+    throw Error("Error sending telegram message");
+  }
+}
+
+// TELEGRAM COMMANDS UTILS
+
+async function sendTelegramMessage({
+  bot,
+  chatId,
+  message
+}: {
+  bot: TelegramBot;
+  chatId: string;
+  message: string;
+}): Promise<void> {
+  await bot.sendMessage(chatId, message, { parse_mode: "Markdown" });
+}
+
+function checkIfChannelIdExists(chatId: string): boolean {
+  const channelIds = db.telegramChannelIds.get();
+  if (channelIds.includes(chatId)) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+function subscribeChannelId(channelId: string): void {
+  const channelIds = db.telegramChannelIds.get();
+  channelIds.push(channelId);
+  db.telegramChannelIds.set(channelIds);
+}
+
+function unsubscribeChannelId(channelId: string): void {
+  const channelIds = db.telegramChannelIds.get();
+  channelIds.splice(channelIds.indexOf(channelId), 1);
```

Comment:
Although correct, there's not need to mutate the data with splice. Just use `.filter()`